src/PublicApiAnalyzers/Core/Analyzers/PublicApiAnalyzerResources.resx

```diff
@@ -160,7 +160,8 @@
     <value>Symbol '{0}' is part of the declared API, but is either not public or could not be found</value>
   </data>
   <data name="RemoveDeletedApiDescription" xml:space="preserve">
-    <value>When removing a public type or member the corresponding entry in PublicAPI.txt should also be removed. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.</value>
+    <value>When removing a public type or member, put that entry in PublicAPI.Unshipped.txt with '*REMOVED*' prefix. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.</value>
+    <note>{Locked="*REMOVED*"}</note>
```

Comment:
Ohhh, this should have been `<comment>` not `<note>`

I haven't confirmed if that fixes the generation, but it should, I can't imagine there is any other issue.

Reply:
🤦‍♂️ I looked at a wrong file.... in xlf files the "comment" is turned into a "note"